docs/src/pages/[platform]/connected-components/liveness/full-api.react.mdx

```diff
@@ -18,10 +18,16 @@ Below is the full list of props that can be used with the `FaceLivenessDetectorC
 
 ### FaceLivenessDetectorConfig
 
-Below is the full list of props that can be used with the `FaceLivenessDetectorConfig`. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts#L42).
+Below is the full list of props that can be used with the `FaceLivenessDetectorConfig`. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/service/types/error.ts). For the error text strings used for translation you can view the [display text file](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts).
 
 <ReactPropsTable props={FACE_LIVENESS_DETECTOR_CONFIG} />
 
+### Error States
+
+Below is the full list of error states that can be returned from the `onError` prop. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts#L102).
```

Comment:
Should this reference the new errors.ts file?

Reply:
Hm... I swear i changed this

Reply:
Fixed!